libs/plpgsql.json

```diff
@@ -0,0 +1,46 @@
+  {"id": "plpgsql.pg-stat-statements", "imports": ["pg_stat_statements"], "name": "pg_stat_statements", "repo": "postgres/postgres/tree/master/contrib/pg_stat_statements", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgstattuple", "imports": ["pgstattuple"], "name": "pgstattuple", "repo": "postgres/postgres/tree/master/contrib/pgstattuple", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-trgm", "imports": ["pg_trgm"], "name": "pg_trgm", "repo": "postgres/postgres/tree/master/contrib/pg_trgm", "tags": ["string-search"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pg-visibility", "imports": ["pg_visibility"], "name": "pg_visibility", "repo": "postgres/postgres/tree/master/contrib/pg_visibility", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.postgres-fdw", "imports": ["postgres_fdw"], "name": "postgres_fdw", "repo": "postgres/postgres/tree/master/contrib/postgres_fdw", "tags": ["db-link"], "tech": ["app-arch"]},
+  {"id": "plpgsql.seg", "imports": ["seg"], "name": "seg", "repo": "postgres/postgres/tree/master/contrib/seg", "tags": ["intervals"], "tech": ["data-structures"]},
+  {"id": "plpgsql.sepgsql", "imports": ["sepgsql"], "name": "sepgsql", "repo": "postgres/postgres/tree/master/contrib/sepgsql", "tags": ["access-control"], "tech": ["security"]},
+  {"id": "plpgsql.spi", "imports": ["spi"], "name": "spi", "repo": "postgres/postgres/tree/master/contrib/spi", "tags": ["spi"], "tech": ["api"]},
+  {"id": "plpgsql.sslinfo", "imports": ["sslinfo"], "name": "sslinfo", "repo": "postgres/postgres/tree/master/contrib/sslinfo", "tags": ["ssl"], "tech": ["security"]},
+  {"id": "plpgsql.tablefunc", "imports": ["tablefunc"], "name": "tablefunc", "repo": "postgres/postgres/tree/master/contrib/tablefunc", "tags": ["plpgsql-c"], "tech": ["interoperability"]},
+  {"id": "plpgsql.tcn", "imports": ["tcn"], "name": "tcn", "repo": "postgres/postgres/tree/master/contrib/tcn", "tags": ["triggers"], "tech": ["utils"]},
+  {"id": "plpgsql.tsm-system-rows", "imports": ["tsm_system_rows"], "name": "tsm_system_rows", "repo": "postgres/postgres/tree/master/contrib/tsm_system_rows", "tags": ["statistics"], "tech": ["computational-science"]},
+  {"id": "plpgsql.tsm-system-time", "imports": ["tsm_system_time"], "name": "tsm_system_time", "repo": "postgres/postgres/tree/master/contrib/tsm_system_time", "tags": ["system-time"], "tech": ["utils"]},
+  {"id": "plpgsql.unaccent", "imports": ["unaccent"], "name": "unaccent", "repo": "postgres/postgres/tree/master/contrib/unaccent", "tags": ["text-search"], "tech": ["computational-science"]},
+  {"id": "plpgsql.uuid-ossp", "imports": ["uuid-ossp"], "name": "uuid-ossp", "repo": "postgres/postgres/tree/master/contrib/uuid-ossp", "tags": ["uuid"], "tech": ["utils"]},
+  {"id": "plpgsql.xml2", "imports": ["xml2"], "name": "xml2", "repo": "postgres/postgres/tree/master/contrib/xml2", "tags": ["xml"], "tech": ["app-date"]}
```

Comment:
app-data, not app-date

Reply:
BTW, does your script (technologies.js) prints warnings for previously unseen categories? Would be very useful.

Reply:
yeah, I was thinking to extend it to make more validation like checking for dupes - copy/paste errors and such